data/discovery/register/citizens-advice-bureau.yaml

```diff
@@ -0,0 +1,13 @@
+fields:
+- citizens-advice-bureau
+- name
+- address
+- citizens-advice-bureau-membership-number
+- citizens-advice-bureau-membership-organisation-type
+- citizens-advice-bureau-contact-region
+- start-date
+- end-date
+phase: discovery
+register: citizens-advice-bureau
+registry: citizens-advice
```

Comment:
`citizens-advice` isn't in [ORJ](https://github.com/openregister/openregister-java/blob/master/src/main/resources/config/public-bodies.yaml) or even the [Whitehall API](https://www.gov.uk/api/organisations).  I think we should make this `cabinet-office` for now.

@michaelabenyohai how can we handle this?